.gitattributes

```diff
@@ -0,0 +1,5 @@
+# Auto detect text files and perform LF normalization
+*       text=auto
```

Comment:
@kevingrillet I'm sorry, but I'll have to let this auto-format... I use [this](https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format) extension, maybe you could use it as well so we have the same type of formatting. It's actually quite smart, because stuff like this:
```sh
function1 # Comment 1
function2 # Comment 2
veryLongFunctionThatHasALongName1 # Comment 3
```
gets formatted into this:
```sh
function1                         # Comment 1
function2                         # Comment 2
veryLongFunctionThatHasALongName1 # Comment 3
```
I hope you'll find it useful.